nmi_tools/glue/build/build_uw_deb.sh

```diff
@@ -90,8 +90,10 @@ elif [ "$VERSION_CODENAME" = 'focal' ]; then
 elif [ "$VERSION_CODENAME" = 'jammy' ]; then
     dch --distribution $dist --nmu 'place holder entry'
     dch --distribution $dist --nmu 'place holder entry'
+elif [ "$VERSION_CODENAME" = 'chimaera' ]; then
+    dch --distribution $dist --nmu 'place holder entry'
```

Comment:
This line is needed to disambiguate between version in the same repository. Since we won't put this into a repository (or it is the only Devuan version we build), I would simply change this line to 'true'. As it is, this line changes the package version number from 10.0.3-1 to 10.0.3-1.1

Reply:
This is what I get for cargo-culting my way through the packaging code. :)